Formula/spotbugs.rb

```diff
@@ -0,0 +1,21 @@
+class Spotbugs < Formula
+  desc "SpotBugs is FindBugs' successor. A tool for static analysis to look for bugs in Java code."
```

Comment:
Let's keep it to one sentence, and no period at the end

Reply:
done